### ISSUE TITLE
Finding Jobs 

### ISSUE THREAD

**Issue by HussienAlasker** [HussienAlasker/Joblang#1]
Are look for a job in United Kingdom. Are you finding the processing of finding job difficult? Sure finding a job that you want is not easy and especially if you do not have a current a job and you are unemployed for a long period of time.

There are two things you should know that some companies do not do job posting and they look at candidates or post in online groups. Therefore, we need to have full understanding what terms and keywords non-technical people are using to search for people that have specific skills in certain area. This task or research requires using Google or Keyword research tools to understand what terms are being used to look for candidates.
So you need to optimize your CV and profiles with these keywords in order to appear under these terms when people search for these specific terms. As a result, your CV should be relevant and contain these terms in order to get index under these terms.

In addition, you need to create profiles in all job boards since some of these job boards have high rank for certain searches than other job boards.
As well, knowing these keywords will help you to find the highest number of opportunities that match your skills and specialization.
In conclusion, you need to optimize your CV with Keywords and perform searches on job boards and search engines using these terms.

About [Joblang](https://joblang.com/job/en):
[Joblang.com](https://joblang.com/job/en) is specifically designed to connect native, bilingual, and multilingual job seekers with the international markets and job engine platforms to find the right recruitment in a full array of genres. [Joblang.com](https://joblang.com/job/en) searches hundreds of websites, platforms, and newspapers, and instantly filters all the potential jobs according to the languages you speak. Simply, [Joblang.com](https://joblang.com/job/en) does the work for you
